manifests/init.pp

```diff
@@ -294,7 +294,8 @@
 ) inherits foreman::params {
   assert_type(Array[Stdlib::IP::Address], $trusted_proxies)
 
-  if $db_sslmode == 'UNSET' and $db_root_cert {
+  $db_database_real = pick($foreman::db_database, 'foreman')
```

Comment:
What's the reason not to default `db_database` to `foreman` in the first place?

Reply:
No good reason, will update.